src/r1cs/vm.rs

```diff
@@ -118,9 +118,11 @@ impl<'a, T: FieldElement> VM<'a, T> {
                     }
                 }
                 AstNode::Rtrn(expr) => {
-                    self.compiler_state
-                        .messages
-                        .insert(0, format!("return call in {}", self.name));
+                    let fn_source_path = self.compiler_state.fn_to_path.get(&self.name).unwrap();
+                    self.compiler_state.messages.insert(
+                        0,
+                        format!("return call in {}() ({})", self.name, fn_source_path),
```

Comment:
```suggestion
                        format!("return call in {}", fn_source_path),
```

The source filename is always the function name. If we show the path we shouldn't need to show the filename as well.